test/new-e2e/pkg/utils/e2e/client/host.go

```diff
@@ -224,6 +224,18 @@ func (h *Host) FileExists(path string) (bool, error) {
 	return info.Mode().IsRegular(), nil
 }
 
+// EnsureFileIsReadable add readable rights to a remote file
+func (h *Host) EnsureFileIsReadable(path string) error {
+	// ensure the file is readable on the remote host
+	if h.osFamily != oscomp.WindowsFamily {
+		_, err := h.Execute(fmt.Sprintf("sudo chmod +r %s", path))
```

Comment:
Can we add a TODO for Windows support?

Reply:
Sure, thought not needed for the flare so far

Reply:
Adding the TODO in a follow up

Reply:
not likely to be necessary on Windows since the SFTP session is authenticated as the `Administrator` user.